lib/routes/token.js

```diff
@@ -16,9 +17,20 @@ const P = require('../promise');
 const Scope = require('../scope');
 const validators = require('../validators');
 
+const HEX_STRING = validators.HEX_STRING;
+
 const MAX_TTL_S = config.get('expiration.accessToken') / 1000;
 const GRANT_AUTHORIZATION_CODE = 'authorization_code';
 const GRANT_REFRESH_TOKEN = 'refresh_token';
+const GRANT_JWT = 'urn:ietf:params:oauth:grant-type:jwt-bearer';
+
+const JWT_AUD = config.get('publicUrl') + '/v1/token';
```

Comment:
>  I don't recall seeing this pattern used elsewhere

It's used in both [1] and [2] so I guess I just haven't been paying attention.  :+1: 

[1] https://developers.google.com/identity/protocols/OAuth2ServiceAccount
[2] http://self-issued.info/docs/draft-ietf-oauth-jwt-bearer.html